src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/services/service-daemon-list/service-daemon-list.component.ts

```diff
@@ -68,8 +80,45 @@ export class ServiceDaemonListComponent implements OnInit, OnChanges, AfterViewI
     private hostService: HostService,
     private cephServiceService: CephServiceService,
     private orchService: OrchestratorService,
-    private relativeDatePipe: RelativeDatePipe
-  ) {}
+    private relativeDatePipe: RelativeDatePipe,
+    public actionLabels: ActionLabelsI18n,
+    private authStorageService: AuthStorageService,
+    private daemonService: DaemonService,
+    private notificationService: NotificationService
+  ) {
+    this.permissions = this.authStorageService.getPermissions();
+
+    this.tableActions = [
```

Comment:
As a recommended practice, we should have only the minimum required logic in the constructors, thus movin the initialization logic to `ngOnInit` when possible.